staging/src/k8s.io/client-go/discovery/fake/discovery.go

```diff
@@ -51,11 +51,6 @@ func (c *FakeDiscovery) ServerResourcesForGroupVersion(groupVersion string) (*me
 }
 
 func (c *FakeDiscovery) ServerResources() ([]*metav1.APIResourceList, error) {
-	action := testing.ActionImpl{
-		Verb:     "get",
-		Resource: schema.GroupVersionResource{Resource: "resource"},
-	}
-	c.Invokes(action, nil)
```

Comment:
what's the reason for the change here?

Reply:
@yliaog I believe these codes are useless here, aren't they? Am I missing sth?

Reply:
i tend to agree with you, however, i don't quite understand why the code for c.Invoke(...) was introduced in the first place, similarly for the other methods ServerResourcesForGroupVersion(), ServerGroups(), ServerVersion()

Reply:
Yeah.. IDK either. But I'm pretty sure it should be cleaned up anyway.

Reply:
Cleaning up for for other methods too :)